rpm_spec/subpackages/manageiq-appliance

```diff
@@ -65,6 +65,7 @@ Requires: cockpit-ws
 # Add sub uids/gids for running non-privileged containers
 grep manageiq /etc/subuid >/dev/null 2>&1 || /usr/sbin/usermod --add-subuids 100000-165535 manageiq
 grep manageiq /etc/subgid >/dev/null 2>&1 || /usr/sbin/usermod --add-subgids 100000-165535 manageiq
+/usr/sbin/usermod -a -G disk manageiq
```

Comment:
Can you add a comment here? Otherwise it looks like it's part of the previous comments.